EasyPost/Utilities/Rates.cs

```diff
@@ -225,66 +225,66 @@ public static EasyPost.Models.API.Beta.StatelessRate GetLowestStatelessRate(IEnu
         }
 
         /// <summary>
-        ///     Get the lowest SmartRate from this list.
+        ///     Get the lowest Smart Rate from this list.
         /// </summary>
         /// <param name="smartRates">List of smartRates to parse.</param>
         /// <param name="deliveryDays">Delivery days to include in the filter.</param>
         /// <param name="deliveryAccuracy">Delivery accuracy to include in the filter.</param>
         /// <returns>Lowest rate matching the filter.</returns>
-        public static EasyPost.Models.API.Smartrate GetLowest(this IEnumerable<EasyPost.Models.API.Smartrate> smartRates, int deliveryDays, EasyPost.Models.API.SmartrateAccuracy deliveryAccuracy) => GetLowestSmartRate(smartRates, deliveryDays, deliveryAccuracy);
+        public static EasyPost.Models.API.SmartRate GetLowest(this IEnumerable<EasyPost.Models.API.SmartRate> smartRates, int deliveryDays, EasyPost.Models.API.SmartRateAccuracy deliveryAccuracy) => GetLowestSmartRate(smartRates, deliveryDays, deliveryAccuracy);
 
         /// <summary>
-        ///     Get the lowest SmartRate from a list of <see cref="EasyPost.Models.API.Smartrate"/>s.
+        ///     Get the lowest Smart Rate from a list of <see cref="EasyPost.Models.API.SmartRate"/>s.
         /// </summary>
         /// <param name="smartRates">List of SmartRates to parse.</param>
         /// <param name="deliveryDays">Delivery days to include in the filter.</param>
         /// <param name="deliveryAccuracy">Delivery accuracy to include in the filter.</param>
         /// <returns>Lowest rate matching the filter.</returns>
-        public static EasyPost.Models.API.Smartrate GetLowestSmartRate(IEnumerable<EasyPost.Models.API.Smartrate> smartRates, int deliveryDays, EasyPost.Models.API.SmartrateAccuracy deliveryAccuracy)
+        public static Models.API.SmartRate GetLowestSmartRate(IEnumerable<Models.API.SmartRate> smartRates, int deliveryDays, Models.API.SmartRateAccuracy deliveryAccuracy)
         {
-            EasyPost.Models.API.Smartrate? lowestSmartRate = null;
+            Models.API.SmartRate? lowestSmartRate = null;
 
-            foreach (EasyPost.Models.API.Smartrate smartRate in smartRates)
+            foreach (Models.API.SmartRate smartRate in smartRates)
             {
                 // smartRate will always have a time in transit, don't need to check for null
-                int? smartRateAccuracy = smartRate.TimeInTransit!.GetBySmartrateAccuracy(deliveryAccuracy);
+                int? smartRateAccuracy = smartRate.TimeInTransit!.GetBySmartRateAccuracy(deliveryAccuracy);
 
                 if (smartRateAccuracy == null)
                 {
-                    // If the smartRate doesn't have a time in transit for the specified accuracy, skip it
+                    // If the Smart Rate doesn't have a time in transit for the specified accuracy, skip it
```

Comment:
I believe there should be no space between these two words.

Reply:
Sounds good, will fix